druid/src/widget/painter.rs

```diff
@@ -107,6 +107,19 @@ impl<T> Painter<T> {
 }
 
 impl<T: Data> BackgroundBrush<T> {
+    /// Request paint if the BackgroundBrush changed.
+    pub fn update(&mut self, ctx: &mut UpdateCtx, old_data: &T, data: &T, env: &Env) {
+        match self {
+            Self::ColorKey(key) => {
+                if ctx.env_key_changed(key) {
+                    ctx.request_paint();
+                }
+            }
+            Self::Painter(p) => p.update(ctx, old_data, data, env),
+            _ => {}
```

Comment:
This is the nittiest thing ever but I think that the "catchall" arm of a `match` should just directly return `()`, instead of implicitly returning `()` through an empty block 😅

```suggestion
            _ => (),
```

Reply:
I also use `()` mostly. I think this was generated by rust-analyzer